### PR TITLE
COMP-325 Add support for Python 3.10

### PR DESCRIPTION
**Pull request by rbrazinskas**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+Version 2.1
+===========
+
+* Add support for Python 3.10
```
Comment (hay-k) [inline CHANGELOG.rst:8]:
```suggestion
* Add support for Python 3.10. `#23 <https://github.com/iqm-finland/iqm-client/pull/23>`_
```

**Comment by hay-k** [comment]
The line `python_requires = ~= 3.9` in `setup.cfg` should be updated too.

**Comment by hay-k** [comment]
The line `python_requires = ~= 3.9` in `setup.cfg` should be updated too.

**Comment by rbrazinskas** [comment]
@hay-k, according to this [https://peps.python.org/pep-0440/#compatible-release](PEP),

~= 3.9 

equals to

>= 3.9, == 3.*

which should cover versions 3.9 and 3.10. Or did I get this wrong?

**Comment by hay-k** [comment]
> @hay-k, according to this [https://peps.python.org/pep-0440/#compatible-release](PEP),
> 
> ~= 3.9
> 
> equals to
> 
> >= 3.9, == 3.*
> 
> which should cover versions 3.9 and 3.10. Or did I get this wrong?

Yes, you are correct, no need to change that one

**Review by hay-k — APPROVED** [review]

**Pull request merged**